docs/docs/connectors/your-own-website.mdx

```diff
@@ -82,7 +82,10 @@ After making the `callback` input available, you can `POST` messages to
 ```json
 {
   "sender": "test_user",  // sender ID of the user sending the message
-  "message": "Hi there!"
+  "message": "Hi there!",
+  "message": {            // optional metadata
```

Comment:
Shouldn't this key be `metadata` instead of `message`?

Reply:
Oops, fixed, thanks.